Tests/UtilityTests/ArgumentParserTests.swift

```diff
@@ -47,7 +47,7 @@ class ArgumentParserTests: XCTestCase {
         let parser = ArgumentParser(commandName:"SomeBinary", usage: "sample parser", overview: "Sample overview")
 
         let package = parser.add(positional: "package name of the year", kind: String.self, usage: "The name of the package")
-        let revision = parser.add(option: "--revision", kind: String.self, usage: "The revision")
+        let revision = parser.add(option: "--revision", kind: String.self, usage: "The revision[Experimental]")
```

Comment:
This can be reverted as it is not experimental :)

Reply:
oops, its mistake.